### DISCUSSION TITLE
Не работает на win 11

### DISCUSSION THREAD

**Discussion by NautroKazenaki** [bol-van/zapret#1013] [Q&A]
У меня одного так или не поддерживается? Тем и вопросов других не видел на этот счёт, в инструкции тоже не нашел инфы

**Comment by ghost** [comment]


**Reply by NautroKazenaki** [reply]
У меня 4 компа, на трёх стоит десятка и там всё идеально, на 11 не работает. Или мне ещё подробнее нужно объяснить, почему я тему создал?

**Comment by 8hrsk** [comment]
Есть готовый билд для винды flowseal/zapret-youtube-discord

Можете использовать его. А так, запрет работает как на *NIX, так и на Dos системах

**Reply by NautroKazenaki** [reply]
увы, всё также не работает

**Reply by bol-van** [reply]
На DOS он не работает. Причем тут DOS ?
Текущие винды - это наследники Windows NT, а не 3.1/95/98

**Reply by 8hrsk** [reply]
Странно. Попробуйте на win11 отключить антивирусы и брэндмауер. Понятия не имею, в чем может быть проблема, но вот лично у меня стоит win11 и все прекрасно работает

**Reply by 8hrsk** [reply]
> На DOS он не работает. Причем тут DOS ?

Новый Год, затупил )

**Comment by TheRigelKing** [comment]
> У меня один такой случай или нет? Темы и вопросы, которые другие не видели на этой странице, в инструкции тоже не нашел инфы

У меня возникла похожая проблема Версия Win11 24H2, на Win 10 все работает идельно 
Что я делаю не так ?

**Reply by TheRigelKing** [reply]
Антивирусы и брэндмауер отключен

**Comment by plumpbeer** [comment]
Killer Network Service, это оно издевательски мешает zapret'у. Уберите из автозагрузок через msconfig. Я 5 раз снёс окна что бы вычислить вредителя и нашел его чисто случайно в порыве бешенства снимая задачи в диспетчере.

**Reply by 8hrsk** [reply]
Удивительная вещь. Ни разу такой проблемы не было. Недавно ставил с нуля шинду последней версии, всё живёт
